lib/cli/index.js

```diff
@@ -56,6 +56,21 @@ module.exports = (options = {}) => {
       silent: 'verbose',
       'log-level': ['silent', 'verbose']
     })
+    .option('canonical', {
```

Comment:
Just a question, this is the equivalent meaning of "--prefer-online" right?  So always fetch?

Reply:
If the above, then what is the difference with `fetch`?   Maybe the subtlety in the difference is confusing.  If it is for me, I guess it will also be for end users.

Reply:
Canonical means use the "canonical" source of truth which is the online version. If you specify that without "fetch" it will show you the URL for the canonical data but won't fetch it unless fetch is specified.  This lets you get info about where the canonical data is without being forced to fetch it.

Once we land this PR my next step will be a PR for doc to provide more info on how to use the commands.

Reply:
Oh right!! I remember now.  I do think this behavior will need good docs because it is a bit confusing.  Thanks for the clarification.